child_compassion/wizards/child_search_attributes.py

```diff
@@ -0,0 +1,124 @@
+
+    @api.multi
+    def send(self):
+
+        holds = self.env['compassion.hold']
+        messages = self.env['gmc.message.pool']
+        global_child = self.env['compassion.childpool.search'].browse(
+            self.env.context.get('active_id'))
+
+        for child in global_child.global_child_ids:
+            # Save children form global children to compassion children
+
+            child_vals = {
+                'global_id': child.global_id,
+                'local_id': child.local_id,
+                'project_id': child.project_id.id,
+                'field_office_id': child.field_office_id,
+                'name': child.name,
+                'firstname': child.firstname,
+                'lastname': child.lastname,
+                'preferred_name': child.preferred_name,
+                'gender': child.gender,
+                'birthdate': child.birthdate,
+                'age': child.age,
+                'is_orphan': child.is_orphan,
+                'beneficiary_state ': child.beneficiary_state,
+                'sponsorship_status': child.sponsorship_status,
+                'unsponsored_since': child.unsponsored_since,
+            }
+            child_comp = self.env['compassion.child'].create(child_vals)
+
+            # Create Holds for children to reserve
+            hold_vals = {
+                'name': self.name,
+                'child_id': child_comp.id,
+                'type': self.type,
+                'expiration_date': self.hold_expiration_date,
+                'primary_owner': self.primary_owner,
+                'secondary_owner': self.secondary_owner,
+                'no_money_yield_rate': self.no_money_yield_rate,
+                'yield_rate': self.yield_rate,
+                'channel': '',
+                'source_code': '',
+            }
+            hold = holds.create(hold_vals)
+            holds += hold
+
+            # create messages to send to Connect
+            action_id = self.env.ref('child_compassion.create_hold').id
+
+            messages += messages.create({
+                'action_id': action_id,
+                'object_id': hold.id
+            })
+        messages.with_context(async_mode=False).process_messages()
+
+        # update compassion children with hold_id received
+        for hold in holds:
+            child_to_update = hold.child_id
+            if hold.hold_id:
+                child_vals = {
+                    'hold_id': hold.id,
+                    'active': True,
+                    'state': 'N',
+                }
+                child_to_update.write(child_vals)
+            else:
+                # delete child if no hold_id received
+                child_to_update.unlink()
+                hold.unlink()
+
+        return {
+            'name': _(''),
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form,tree',
+            'res_model': 'compassion.hold',
+            'target': 'current',
```

Comment:
If you add
```
'domain': [('id', 'in', holds.ids)]
````
It will only display the freshly created holds rather than all existing holds.